examples/auction/src/lib.rs

```diff
@@ -150,6 +158,15 @@ fn view<'a, 'b, S: HasStateApi>(
     Ok(host.state())
 }
 
+/// ViewBalance function that returns the balance of the contract
+#[receive(contract = "auction", name = "viewBalance", return_value = "Amount")]
```

Comment:
Off-chain you can already get the balance of a smart contract.
But this could be useful for another smart contract, I guess.
Another contract might be more interested in the current highest bid and not the balance, so in this case I would say it should be renamed.